synapse/lib/hiveauth.py

```diff
@@ -1067,6 +1067,17 @@ async def genGateInfo(self, gateiden):
         return info
 
     def confirm(self, perm, default=None, gateiden=None):
+        # PROP_SET_PREFIX = ('node', 'prop', 'set')
```

Comment:
This is WIP. I was thinking we could do the checks dynamically but from `<user>.confirm()`, we don't have access to the cortex so I'm not sure this will work.